### PR TITLE
moderation-policy: clarification RE recertification

### PR DESCRIPTION
**Pull request by refack**
Since this year will be the first time we do recertification, it's a good opportunity to better define what this entails.

### CODE REVIEW

**Comment by refack** [comment]
A few questions raised in the CommComm meeting:
* Do we want to do this as an individual's performance review (activity, availability, policy), or as a general approval of the mod-team for the last year?
* Do we have enough information to properly evaluate?

File: Moderation-Policy.md [hunk 1]
```diff
@@ -252,16 +252,20 @@ recertification. If there are no objections after seven days, the nomination is
...
 New Moderation Team members are onboarded with:
 - an invite to Node.js Moderation Team Slack
 - permission changes made to GitHub to allow access to moderate
 - a walkthrough of relevant processes, expectations, and documents by an existing Moderation Team member
 - access to existing documents
 
+*Recertification*
+
+An annual recertification vote is required for all Moderation Team members.
+For an individual to be recertified, a simple majority vote in *favor* of recertification
```
Comment (Trott) [inline Moderation-Policy.md]: Nit: trailing space

Comment (Trott) [reply Moderation-Policy.md]: Nit: wrap at 80 chars

**Comment by Trott** [comment]
Answers are my personal opinion only and not based on any previously-reached consensus or discussion with others:

> * Do we want to do this as an individual's performance review (activity, availability, policy), or as a general approval of the mod-team for the last year?

Individual performance. More than that, trust. IMO, someone would have to do some pretty egregious stuff to get flagged as untrusted, but it could happen. Even if it ends up being mostly a rubber stamp, I like recertification because it guarantees the Moderation Team has the trust of both CommComm and TSC. (It requires a rejection by both CommComm and TSC to be removed from the Moderation Team prior to recertification, but it requires a rejection of only one or the other to be removed during annual recertification.)

* Do we have enough information to properly evaluate?

Sort of. There's a lot that's hidden because Moderation Team discussions are usually private. Pieces of information that folks might have that are useful:

* Activity in the moderation repo and the moderation actions documented there.
* Streamed/recorded Moderation Team meetings and minutes.

That may be enough?

**Comment by mhdawson** [comment]
I see the annual review as a checkpoint to reflect on if there have been any complaints, visible issues over the past year with respect to how the individuals have engaged.  As @Trott says it is basically a checkpoint to confirm that the members still have the required trust.  As such I don't think we need any more information/process etc.

**Comment by Trott** [comment]
It's also a good reminder to periodically review our Moderation Policy and adjust it to improve it (like this PR).

**Comment by bnb** [comment]
I will review today 👍

**Pull request merged**